notification/src/config/config.js

```diff
@@ -111,7 +112,7 @@ function loadAndValidateConfig() {
 
 loadAndValidateConfig()
 
-module.exports = {
+export default {
```

Comment:
Why do you need default here?

Reply:
In few cases where the export needs to be an object, I am using default. (So I am importing config.js as object)
If you have any better way let me know.

Reply:
Can you add this comment to the code?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/943/commits/9ec1910738df93f3412731231b34890e15733697